### PR TITLE
Streamline SciPy Begins section

### PR DESCRIPTION
**Pull request by mdhaber**
Partially addresses #222.

While I wish we could include the entire background section, here I've tried to streamline the SciPy Begins section. Please see self-review for rationale.

This might fully address #222, actually. There aren't large swaths in the rest of the background that I'd consider removing, except maybe the paragraph about NumFOCUS.

### CODE REVIEW

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -352,45 +352,6 @@ \subsection*{SciPy Begins}
 the Space Telescope Science Institute (STScI), which was in charge of
 Hubble Space Telescope science operations, decided to replace their
 custom scripting language and analysis pipeline with Python\cite{STScI-slither}.
-
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: This section is great, but it's not part of the history of SciPy 1.0.

File: scipy-1.0/paper.tex [hunk 2]
```diff
@@ -352,45 +352,6 @@ \subsection*{SciPy Begins}
...
-As SciPy, the algorithms library of the ecosystem, began attracting the attention
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: This section is great, but it's not really part of the history of SciPy 1.0. If we have to trim, this seems a natural place to start.

File: scipy-1.0/paper.tex [hunk 3]
```diff
@@ -401,55 +362,9 @@ \subsection*{SciPy Begins}
...
-%to the NumPy mailing list:
-%\begin{quote}
-%IEEE's magazine, Computing in Science and Engineering (CiSE), has asked me
-%to put together a theme issue on the use of Python in Science and
-%Engineering. I will write an overview to be accompanied by 3-5 articles [. . .]
-%that show a diverse set of applications or
-%tools, to give our readers a sense of whether or not Pyth%on might be useful
-%in their own work.
-%\end{quote}
-In a May/June 2007 special issue of IEEE Computing in Science and
```
Comment (mdhaber) [inline scipy-1.0/paper.tex]: The special issue is mentioned in "SciPy Matures". If there are really important parts of this that we want to keep, we can add them back there.

**Comment by mdhaber** [comment]
Before these changes, the Background section was 2000 words and took 3 pages. The SciPy Begins subsection was 850 words and took about 1 page (plus a 1/3 page figure). This PR would remove 324 words (~38%) from the SciPy begins subsection; the subsection would now take approximately 1/2 page (plus the 1/3 page figure).

**Comment by tylerjereddy** [comment]
Alright, merging with Jarrod's approval

**Pull request merged**